<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AgentWindow</class>
 <widget class="QWidget" name="AgentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="serviceNameLineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="descriptionLineEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Название услуги:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Описание услуги:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="serviceDateEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="serviceTimeEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>118</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата услуги:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Время услуги:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="capacityLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>140</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Вместимость: </string>
   </property>
  </widget>
  <widget class="QTableWidget" name="serviceTableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Список услуг</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addServiceButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>50</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить услугу </string>
   </property>
  </widget>
  <widget class="QPushButton" name="editServiceButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>100</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить услугу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteServiceButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить услугу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exportBookingsButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>240</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Выгрузить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
